changes/256.bugfix.rst

```diff
@@ -1,2 +1,10 @@
 Retain Objective-C objects when creating Python wrappers and release them when the
-Python wrapped is garbage collected.
+Python wrapped is garbage collected. This means that manual ``retain`` calls and
+subsequent ``release`` or ``autorelease`` calls from Python are no longer needed with
```

Comment:
I think this change note counts as a "feature" rather than a "bugfix".

Reply:
Done.